Index: tuya_sdk-1.0/platform/posix/CMakeLists.txt
===================================================================
--- tuya_sdk-1.0.orig/platform/posix/CMakeLists.txt
+++ tuya_sdk-1.0/platform/posix/CMakeLists.txt
@@ -1,7 +1,7 @@
 include( ${LIBRARIES_DIR}/mbedtlsFilePaths.cmake )
 
 # Create target for POSIX implementation of reconnect logic.
-add_library( platform_port STATIC
+add_library( platform_port
                "mbedtls_sockets_wrapper.c"
                "network_wrapper.c"
                "system_wrapper.c"
