Index: tuya_sdk-1.0/middleware/CMakeLists.txt
===================================================================
--- tuya_sdk-1.0.orig/middleware/CMakeLists.txt
+++ tuya_sdk-1.0/middleware/CMakeLists.txt
@@ -2,7 +2,7 @@ include( ${LIBRARIES_DIR}/coreHTTP/httpF
 include( ${LIBRARIES_DIR}/coreMQTT/mqttFilePaths.cmake )
 
 # Create target for POSIX implementation of reconnect logic.
-add_library( middleware_implementation STATIC
+add_library( middleware_implementation
                 ${MQTT_SOURCES}
                 ${MQTT_SERIALIZER_SOURCES}
                 ${HTTP_SOURCES}
